.github/workflows/deploy.yaml

```diff
@@ -20,14 +19,13 @@ jobs:
       - name: Checkout Docs
         run: git fetch && git checkout ${GITHUB_SHA}
```

Comment:
I wasn't sure if we wanted to do `git checkout ${GITHUB_SHA}` or `git checkout master`